NuGet-VS.slnf

```diff
@@ -0,0 +1,21 @@
+{
+  "solution": {
+    "path": "NuGet.sln",
+    "projects": [
+      "src\\NuGet.Clients\\NuGet.PackageManagement.PowerShellCmdlets\\NuGet.PackageManagement.PowerShellCmdlets.csproj",
+      "src\\NuGet.Clients\\NuGet.PackageManagement.UI\\NuGet.PackageManagement.UI.csproj",
+      "src\\NuGet.Clients\\NuGet.PackageManagement.VisualStudio\\NuGet.PackageManagement.VisualStudio.csproj",
+      "src\\NuGet.Clients\\NuGet.Tools\\NuGet.Tools.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio.Client\\NuGet.VisualStudio.Client.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio.Common\\NuGet.VisualStudio.Common.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio.Contracts\\NuGet.VisualStudio.Contracts.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio.Implementation\\NuGet.VisualStudio.Implementation.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio.Internal.Contracts\\NuGet.VisualStudio.Internal.Contracts.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio.Interop\\NuGet.VisualStudio.Interop.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio.OnlineEnvironment.Client\\NuGet.VisualStudio.OnlineEnvironment.Client.csproj",
+      "src\\NuGet.Clients\\NuGet.VisualStudio\\NuGet.VisualStudio.csproj",
+      "src\\NuGet.Core\\NuGet.Common\\NuGet.Common.csproj",
```

Comment:
What if I modify something on NuGet.Protocol and I want to test it in NuGet PM UI ?

Reply:
use nuget-src.slnf instead?

Reply:
Actually, `NuGet.Common does seem out of place in this filter, since it's the only `src\NuGet.Core` project. Should it be removed?

Reply:
I think what you ar elooking for at 17 projects, basically everything in NuGet.Clients *except* NuGet.Commandline.

Reply:
16 projects 😁. NuGet.MSSigning is also not VS related (it's for package signing for package repositories?)

Reply:
Good catch.

Reply:
It's a specially built exe used for signing by ESRP.

Reply:
> Actually, `NuGet.Common` does seem out of place in this filter, since it's the only `src\NuGet.Core` project. Should it be removed?

I chose to leave it because I thought it might be handy to have utilities available to change. Looking over them, it may not be common (pun intended) to change any of those, though.... I'm indifferent!

Reply:
16 freshly sorted projects hot off the presses!